apps/system/style/statusbar/statusbar.css

```diff
@@ -50,10 +50,8 @@
   visibility: visible;
 }
 
-#screen:not(.locked):not(.minimized-tray):not(.rocketbar-focused):not(.utility-tray).fullscreen-app #statusbar,
-#screen:not(.locked):not(.minimized-tray):not(.rocketbar-focused):not(.utility-tray).fullscreen-layout-app #statusbar,
-#screen.locked.secure-app:not(.minimized-tray):not(.rocketbar-focused):not(.utility-tray).fullscreen-app #statusbar,
-#screen.locked.secure-app:not(.minimized-tray):not(.rocketbar-focused):not(.utility-tray).fullscreen-layout-app #statusbar {
```

Comment:
Why are removing the `secure-app` rules here? This breaks the lockscreen camera (ie. it will show the statusbar when lockscreen passcode is enabled, and you go to the camera).